chart/templates/uds-package.yaml

```
@@ -11,19 +11,34 @@ spec:
       redirectUris:
         - "https://{{ .Values.subdomain }}.{{ .Values.domain }}/*"
       defaultClientScopes:
-        - "openid"
+        {{ if eq .Values.sso.protocol "gitlab" }}
```

```suggestion
        - "openid"
        {{ if eq .Values.sso.protocol "gitlab" }}
```

Sorry I wasn't explicit - openid always needs to be there.